assets/src/scripts/components/Cards/Versions.tsx

```diff
@@ -79,7 +79,7 @@ function Version({ version }: { version: VersionProps }) {
 export default function Versions() {
   const versions: VersionProps[] = readValueFromPage("versions");
 
-  if (!versions) return null;
+  if (!versions?.length) return null;
 
```

Comment:
If `versions` is an empty array, then `!versions` is false, but `!versions?.length` is true - was that an intentional change?

Reply:
Gah, that breaks the old tests 🤦‍♂️ I'll leave it in there for the moment

Reply:
Fine. I was just checking the change in behaviour was intentional - and it was, so great.